.github/workflows/onmerge.yml

```diff
@@ -0,0 +1,50 @@
+name: Runtimes Onmerge
+on:
+  push:
+    branches:
+      - master
+jobs:
+  build-golang-onmerge:
+    runs-on: ubuntu-latest
+    env:
+      DOCKER_USERNAME: ${{ secrets.DOCKER_USERNAME }}
+      DOCKER_PASSWORD: ${{ secrets.DOCKER_PASSWORD }}
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - run: |
+          cd ./stable/golang
+          make build-all
+          docker login -u="$DOCKER_USERNAME" -p="$DOCKER_PASSWORD"
+          images=( 
+            'go:1.14'
```

Comment:
Is this the version of golang? It's very old

Reply:
https://jira-eng-sjc11.cisco.com/jira/browse/ISIWO-1153 created by Mohsin to track this upgrade.